src/scss/layout/_header.scss

```diff
@@ -1,3 +1,61 @@
-.header {
-  height: 90px;
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
 }
+
+body{
+    max-width: 1440px;
+    margin: auto;
+}
```

Comment:
remove these styles in the body. Instead, use the class 'component' in the header element. The class 'component' will implement the layout with the min-width of 1440px.